internal/request/http.go

```diff
@@ -12,9 +12,18 @@ import (
 	authenticationv1 "k8s.io/api/authentication/v1"
 	authorizationv1 "k8s.io/api/authorization/v1"
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	"k8s.io/apiserver/pkg/authentication/user"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var defaultServiceAccountGroups = []string{
```

Comment:
Can we avoid this global variable and instead explicitly add them at line 122?
I think this is expected behaviour with logic that is beyond the internal logic of the HTTP request package, instead is related to the authentication modules of Kubernetes.
Just my 2 cents.